trollflow2/plugins/__init__.py

```diff
@@ -68,8 +68,10 @@ class AbortProcessing(Exception):
 def create_scene(job):
     """Create a satpy scene."""
     defaults = {'reader': None,
-                'reader_kwargs': None,
-                'ppp_config_dir': None}
+                'reader_kwargs': None}
+    from satpy.version import version as satpy_version
```

Comment:
This should be moved to module level and mocked in the tests so that we don't need to actually install Satpy.

Reply:
I moved it to the module level.

I guess the `try:...except:...` counts as a mock, or do you mean actually via the mock module?